files/nrpe/check_object.py

```diff
@@ -144,6 +170,56 @@ def execute(self):
     except:
       raise
 
+class SwiftPublicAvailability():
+  '''
+  Check Swift API call length by listing contents of a public bucket
+  '''
+  options = dict()
+
+  def __init__(self, options):
+     self.options = options
+     if options.s3_host is None:
```

Comment:
As discussed, a Swift function shouldn't reference S3 things unless there is some exceptionally good reason to do so

Reply:
I agree. Thanks.